language/predefined/assertionerror.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+<!-- EN-Revision: 3fc56d76de1007a3c4f60b143c60c071f32546fb Maintainer: samesch Status: ready -->
+<!-- Reviewed: no -->
+<phpdoc:exceptionref xml:id="class.assertionerror" xmlns:phpdoc="http://php.net/ns/phpdoc" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude">
+ <title>AssertionError</title>
+ <titleabbrev>AssertionError</titleabbrev>
+
+ <partintro>
+
+<!-- {{{ Error intro -->
+  <section xml:id="assertionerror.intro">
+   &reftitle.intro;
+   <para>
+    <ooclass><classname>AssertionError</classname></ooclass> wird ausgelöst,
+    wenn eine über <function>assert</function> gemachte Zusicherung
```

Comment:
Not quite sure about this one: 

```suggestion
    wenn eine Bedingung mit <function>assert</function>
```

Reply:
It's a special condition which raises an assertion error. IIRC @cmb69 used this and after I replaced it with "Assertion" I reverted it lately.